pkg/datapath/linux/requirements.go

```diff
@@ -138,8 +139,7 @@ func CheckMinRequirements() {
 
 		// VTEP integration feature requires kernel 1m large instruction support
 		if option.Config.EnableVTEP {
-			supportedMisc := probeManager.GetMisc()
-			if !supportedMisc.HaveLargeInsnLimit {
+			if err := features.HaveLargeInstructions(); err != nil {
```

Comment:
`err` is unused outside of the nil check:
```suggestion
			if features.HaveLargeInstructions() != nil {
```